### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
### Description

Found via `typos --hidden --format brief` and
`codespell -H -S ./docs/assets,*.json -L ines,tthe,nore`

### Checklist

- [x] I have read [the contributing guide](https://github.com/datnguye/dbterd/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [ ] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have opened an issue to add/update docs, or docs changes are not required/relevant for this PR

### CODE REVIEW

File: docs/assets/css/syntax-highlighting.css
Comment (datnguye) [inline docs/assets/css/syntax-highlighting.css:1]: This is awesome tool, thanks for suggesting this! I have to check it out 🫶

However, can you help to exclude css dir, for example this case? Thanks

File: samples/dbt-constraints/1.3/manifest.json
Comment (datnguye) [inline samples/dbt-constraints/1.3/manifest.json:1]: please help to exclude /samples dir as well, thanks

File: mkdocs.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-site_name: DaaC from dbt artifacts
+site_name: DataC from dbt artifacts
```
Comment (datnguye) [inline mkdocs.yml:1]: Oops sorry I missed this one, can you help to preserve it here?

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**